docs/release-notes/0.1/release-0.1.md

```diff
@@ -22,7 +22,7 @@ Or from within Visual Studio's NuGet package manager.
 
 This initial release contains core ML.NET components for enabling machine learning pipelines:
 
-* ML Data Structures (e.g. `IDataView`, `LearningPipeline`)
+* ML Data Structures (for example, `IDataView`, `LearningPipeline`)
```

Comment:
>for example [](start = 22, length = 11)

Live documentation seems *fine*, I guess. **This** however I'm not sure about. These are release notes that have already been released. They're already out.

Reply:
Do you want me to revert in those cases @TomFinley?

Reply:
Yes I think.

---
In reply to: [222494637](https://github.com/dotnet/machinelearning/pull/1016#discussion_r222494637) [](ancestors = 222494637)